anms-ui/public/app/components/management/agents/AgentsManageModal.vue

```diff
@@ -0,0 +1,162 @@
+<template>
+  <div>
+    <b-modal id="manageAgentModal"
+      ref="manageAgentModal"
+      size="xl"
+      @hide="closeModal"
+      title="Manage Agents"
+      hide-footer>
+      <div v-if="agents">
+        <b-row>
+          <b-col cols="3">
+            <div class="agent-info">
+              <div v-for="agent in agents">
+                <b-badge pill
+                  variant="primary"> {{ agent.agent_id_string }} </b-badge>
+              </div>
+            </div>
+            <div v-if="this.ariString">
+              <h5>ARI String:</h5>
```

Comment:
Use the term "text" in the UI for this ARI form instead of "string". I don't know if this same term comes up elsewhere. This only applies to human facing UI, doesn't matter for internal naming.